packages/ra-ui-materialui/src/layout/UserMenu.tsx

```diff
@@ -122,4 +122,13 @@ UserMenu.defaultProps = {
     icon: <AccountCircle />,
 };
 
+export interface UserMenuProps {
+    children?: React.ReactNode;
+    classes?: object;
+    className?: string;
+    label?: string;
+    logout?: React.ReactNode;
+    icon?: ReactNodeLike;
```

Comment:
Why is this one a ReactNodeLike, while children and logout, which also expect a node, are React.ReactNode? What's the difference?

Reply:
I have no idea, I checked some other source from RA, what they use by icon.

Reply:
React-admin doesn't use ReactNodeLike. 

And if you open a PR with code you don't understand, I don't want to merge it.